O/ONNXRuntime/build_tarballs.jl

```diff
@@ -90,23 +101,22 @@ function platform_exclude_filter(p::Platform)
     Sys.isfreebsd(p)
 end
 platforms = supported_platforms(; exclude=platform_exclude_filter)
-platforms = expand_cxxstring_abis(platforms; skip=!Sys.islinux)
 
-cuda_platforms = Platform[]
-for cuda_version in cuda_versions
-    cuda_tag = "$(cuda_version.major).$(cuda_version.minor)"
-    for p in [
-        Platform("x86_64", "Linux"; cuda = cuda_tag),
-        Platform("x86_64", "Windows"; cuda = cuda_tag)
-    ]
-        push!(platforms, p)
-        push!(cuda_platforms, p)
+let cuda_platforms = CUDA.supported_platforms(min_version=v"10.2", max_version=v"11")
```

Comment:
Most people are on CUDA 12 now, so it'd be good to also bump the version and include all supported versions.

Reply:
Yes - will do a modern version in a follow up.